mgmt/rest/fixtures/fixtures.go

```diff
@@ -1,166 +0,0 @@
-// +build legacy small medium large
```

Comment:
Probably. I was working with code from Joel. The code that was in fixtures.go was put into mock_metric_manager.go and other classes in the fixtures package. Should I overwrite the first commit since it isn't really relevant to how the code turned out in the end?